frigate/events.py

```diff
@@ -147,14 +158,15 @@ def run(self):
 
 
 class EventCleanup(threading.Thread):
-    def __init__(self, config: FrigateConfig, stop_event):
+    def __init__(self, config: FrigateConfig, stop_event: Event):
```

Comment:
stop_event is a multiprocessing event, I think this would make this typed as the Frigate Event db model

Reply:
I think you are right.
The ambiguity is kind of dangerous...
See:
https://github.com/blakeblackshear/frigate/blob/1d45b0b3519dc7ce2e3c3a11747c61aa52605efc/frigate/app.py#L5
https://github.com/blakeblackshear/frigate/blob/1d45b0b3519dc7ce2e3c3a11747c61aa52605efc/frigate/app.py#L28
https://github.com/blakeblackshear/frigate/blob/1d45b0b3519dc7ce2e3c3a11747c61aa52605efc/frigate/app.py#L48

Reply:
What do you think, should we rename the models to FrigateEvent and FriagteRecordings?

Reply:
Yeah, that is ambiguous. I was thinking since the Frigate Event and Recordings are much more common, it may be more straightforward to alias the other `Event` types

Reply:
It seems like `from multiprocessing.synchronize import Event` may just be used for typing, could be `as ProcessEvent` or something 🤔